agenta-web/src/components/TestSetTable/TestsetTable.tsx

```diff
@@ -115,6 +115,31 @@ const TestsetTable: React.FC<testsetTableProps> = ({mode}) => {
         setSelectedRow(gridRef?.current?.getSelectedNodes())
     }
 
+    const handleExportClick = () => {
+        const csvData = convertToCsvFormat(rowData, columnDefs);
+        const filename = `${testsetName}.csv`;
+        downloadCsv(csvData, filename);
+        // Here, you can add the actual CSV export logic later on.
+    };
+    const convertToCsvFormat = (data, columns) => {
+        const header = columns.map(col => col.field).join(",");
+        const rows = data.map(row => 
+            columns.map(col => row[col.field]).join(",")
+        ).join("\n");
+        return `${header}\n${rows}`;
+    };
+    
+    const downloadCsv = (csvContent, filename) => {
```

Comment:
Also here, please add types where required.

Can you also please move this function and convertToCsv one to the `utils.ts` file? Since, they are generic in nature, they can potentially be reused in future.